testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -897,23 +900,34 @@ public void testViewMetadata()
 
         // test SHOW CREATE VIEW
         String expectedSql = formatSqlText(format(
-                "CREATE VIEW %s.%s.%s AS %s",
+                "CREATE VIEW %s.%s.%s SECURITY %s AS %s",
                 getSession().getCatalog().get(),
                 getSession().getSchema().get(),
                 viewName,
+                securityClauseInShowCreate.name(),
                 query)).trim();
 
         actual = computeActual("SHOW CREATE VIEW " + viewName);
 
         assertEquals(getOnlyElement(actual.getOnlyColumnAsSet()), expectedSql);
 
-        actual = computeActual(format("SHOW CREATE VIEW %s.%s." + viewName, getSession().getCatalog().get(), getSession().getSchema().get()));
+        actual = computeActual(format("SHOW CREATE VIEW %s.%s.%s", getSession().getCatalog().get(), getSession().getSchema().get(), viewName));
 
         assertEquals(getOnlyElement(actual.getOnlyColumnAsSet()), expectedSql);
 
         assertUpdate("DROP VIEW " + viewName);
     }
 
+    @DataProvider
+    public static Object[][] testViewMetadataDataProvider()
+    {
+        return new Object[][] {
+                {"", DEFINER},
+                {" SECURITY DEFINER", DEFINER},
+                {" SECURITY INVOKER", INVOKER}
```

Comment:
nit: add a coma at the end:
```
        {" SECURITY INVOKER", INVOKER},
```

`nit` means very small comment that does not require code change, up to you.

Reply:
Sure, makes sense (I like this quirk of Java's grammar, in fact)